src/main/java/net/rubygrapefruit/platform/file/FileWatcher.java

```diff
@@ -11,7 +13,7 @@
  */
 @NotThreadSafe
 public interface FileWatcher {
-    void startWatching(Collection<File> paths);
+    void startWatching(Collection<File> paths) throws InsufficientResourcesForWatchingException;
```

Comment:
Why mention a runtime exception here? And why this specifically, when we can also throw `NativeException` in general, too.

Reply:
The mention is for documentation purposes only. I added the `InsufficientResourcesForWatchingException` since that is something you may want to handle separately for this method.